lib/taiko.js

```diff
@@ -374,13 +374,15 @@ module.exports.switchTo = async targetUrl => {
  *
  * @param {string} requestUrl request URL to intercept
  * @param {function|Object} option action to be done after interception. For more examples refer to https://github.com/getgauge/taiko/issues/98#issuecomment-42024186
- *
+ * @param {number} count number of times the request has to be intercepted . Optional parameter
+ * 
  * @returns {Promise}
  */
-module.exports.intercept = async (requestUrl, option) => {
+module.exports.intercept = async (requestUrl, option, count) => {
```

Comment:
It would be good to add an example with count.

Reply:
Sure @Apoorva-GA i will add them